### PR TITLE
Properties of SETᴰ

### PR DESCRIPTION
**Pull request by hejohns**
#62 wip

### CODE REVIEW

File: Cubical/Categories/Displayed/Fibration/More.agda [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    !/C .over .F-obᴰ _ = tt
+    !/C .over .F-homᴰ _ = tt
+    !/C .over .F-idᴰ = refl
+    !/C .over .F-seqᴰ _ _ = refl
+    !/C .preserves-cartesian _ _ _ _ _ _ _ _ =
+        uniqueExists tt (isPropUnit tt tt) (λ _ p q → isSetUnit tt tt p q) λ _ _ → isPropUnit tt tt
+
+-- fibered terminal objects
+-- TODO: show this gives terminal object in each fiber of the displayed category, directly
+module _ {C : Category ℓC ℓC'} (p : Fibration C ℓCᴰ ℓCᴰ') where
```
Comment (maxsnew) [inline Cubical/Categories/Displayed/Fibration/More.agda:56]: The fibration assumption is never used, just the fact that it’s a functor

File: Cubical/Categories/Displayed/Instances/Sets/Properties.agda [hunk 2]
```diff
@@ -38,3 +38,20 @@ AllCartesianOversSETᴰ {c = A} {A'} B' f .isCartesian {A''} B'' g gfᴰ =
...
   (AllCartesianOversSETᴰ _ _)
+
+open import Cubical.Categories.Displayed.Fibration.More
+open import Cubical.Data.Unit.Base
+open import Cubical.Data.Unit.Properties
+
+hasFiberedTerminalSet : hasFibTerminal isFibrationSet
+hasFiberedTerminalSet dᴰ .vertexᴰ _ = Unit* , isSetUnit*
+hasFiberedTerminalSet dᴰ .elementᴰ = tt
+hasFiberedTerminalSet dᴰ .universalᴰ .equiv-proof unit =
```
Comment (maxsnew) [inline Cubical/Categories/Displayed/Instances/Sets/Properties.agda:49]: Use `uniqueExists`

File: Cubical/Categories/Displayed/Instances/Sets/Properties.agda [hunk 3]
```diff
@@ -38,3 +38,47 @@ AllCartesianOversSETᴰ {c = A} {A'} B' f .isCartesian {A''} B'' g gfᴰ =
...
+open UniversalElement
+setHasTerm : Terminal' (SET ℓ)
+setHasTerm .vertex = Unit* , isSetUnit*
+setHasTerm .element = tt
+setHasTerm .universal X .equiv-proof y = uniqueExists (λ _ → tt*) (isPropUnit tt tt)
+  (λ _ p' q' → isSetUnit tt tt p' q') λ _ _ → funExt λ _ → isPropUnit* tt* tt*
+
+open import Cubical.Categories.Displayed.Limits.Terminal
+
+SETᴰHasTermᴰ : ∀{ℓ ℓ'} → Terminalᴰ (SETᴰ ℓ ℓ') setHasTerm
```
Comment (maxsnew) [inline Cubical/Categories/Displayed/Instances/Sets/Properties.agda:80]: Closest I could find is `Cubical.Categories.Instances.Sets` has a proof that `SET` is complete

**Comment by hejohns** [comment]
closing, since #62 was implemented by #63 #94 #96

**Pull request closed without merging**